app/services/account_creation/device_profile/verification_request.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module AccountCreation
+  module DeviceProfile
+    class VerificationRequest < Proofing::LexisNexis::Ddp::VerificationRequest
+      private
+
+      def build_request_body
+        {
+          api_key: config.api_key,
+          org_id: config.org_id,
+          account_email: applicant[:email],
+          event_type: 'ACCOUNT_CREATION',
+          policy: IdentityConfig.store.authentication_tmx_policy, # Need a sep policy from proofing
```

Comment:
Naming-wise, I might expect this configuration to follow more closely to the existing name `lexisnexis_threatmetrix_policy`.

Maybe something like:

```suggestion
          policy: IdentityConfig.store.lexisnexis_authentication_threatmetrix_policy, # Need a sep policy from proofing
```